tests/unit/store/settings.spec.js

```diff
@@ -374,8 +377,8 @@ describe("store/settings", () => {
       current_focus_customerid
     ); /*  (Save ID) selected */
 
-    let current_customerids = store.getters["settings/customer_account_ids"];
-    let modified_customerids = store.getters["settings/customer_details"];
+    const current_customerids = store.state.settings.customer_account_ids;
+    const modified_customerids = store.state.settings.customer_account_ids[1];
```

Comment:
I think `customer_details` is a useful getter, rather than always having to remember to access the first element in `customer_account_ids`